.clang-format

```diff
@@ -158,7 +170,7 @@ PenaltyReturnTypeOnItsOwnLine: 60
 PenaltyIndentedWhitespace: 0
 PointerAlignment: Right
 ReflowComments: false
-SortIncludes: false
+SortIncludes: CaseInsensitive
```

Comment:
This materially changes the code, just don't do it.

``` 
https://stackoverflow.com/questions/37927553/can-clang-format-break-my-code#:~:text=Short%20answer%3A%20YES.%20The%20clang-format%20tool%20has%20a,of%20existing%20code%2C%20and%20may%20break%20existing%20code
```

Reply:
I'd like to pick an order and make it consistent.  We can shake out initial bugs (which will most likely be simple dependency issues), and after that there should actually be fewer bugs if we are consistent.

LLVM sorts include files (https://llvm.org/docs/CodingStandards.html#include-style).  If it works for them, it can work for us.

Reply:
I think it should be case sensitive, but since we don't have any capitalized headers, I think, it won't matter in practice.